packages/data/src/test/registry.js

```diff
@@ -680,6 +695,27 @@ describe( 'createRegistry', () => {
 			registry.dispatch( 'counter' ).increment( 4 ); // state = 5
 			expect( store.getState() ).toBe( 5 );
 		} );
+
+		it( 'should work with the store object as param for dispatch', async () => {
+			const STORE_NAME = 'demo';
+			const store = registry.registerStore( STORE_NAME, {
+				reducer( state = 'OK', action ) {
+					if ( action.type === 'UPDATE' ) {
+						return 'UPDATED';
+					}
+					return state;
+				},
+				actions: {
+					update() {
+						return { type: 'UPDATE' };
+					},
+				},
+			} );
+
+			expect( store.getState() ).toBe( 'OK' );
+			await registry.dispatch( STORE_NAME ).update();
```

Comment:
This test needs to be updated, to work with the store definition.

Reply:
I left this is as because it was using the redux store directly to access state.